build_runner/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.4.0
+
+- Add a `run` command to execute VM entrypoints from compiled kernel.
```

Comment:
Does it actually support running from kernel yet? I didn't think it did, although it should be trivial to add if you want to add that here?

Reply:
Ah that's right, we never added that though we had discussed it... Updated the changelog entry.